charts/datasaur/templates/config/_image-pull-secret.tpl

```diff
@@ -1,5 +1,5 @@
 {{- define "imagePullSecret" }}
-{{- printf "{\"auths\": {\"%s\": {\"auth\": \"%s\"}}}" .Values.image.registry (printf "%s:%s" .Values.imageCredentials.username .Values.imageCredentials.password | b64enc) | b64enc }}
+{{- printf "{\"auths\": {\"%s\": {\"auth\": \"%s\"}}}" (default .Values.image.registry .Values.imageCredentials.registry) (printf "%s:%s" .Values.imageCredentials.username .Values.imageCredentials.password | b64enc) | b64enc }}
```

Comment:
before this PR, have we supported changing image registry? 🤔  why do you need to add the registry again here? @michael-jong-ds

Reply:
ini ada issue kemarin..
Jadi .Values.image.registry itu kan <aws-id>.us.aws.etc/datasaur/test
Nah tapi credentialsnya itu harus di assign ke <aws-id>.us.aws.etc saja tanpa /datasaur/test, jadi aku saranin mike buat dipisah @tonoman3g

Reply:
this is to pull the image from the provided repository in values.yaml, or if the value is not there, use the default image registry